Palace/AppInfrastructure/TPPConfiguration+SE.swift

```diff
@@ -44,10 +44,14 @@ extension TPPConfiguration {
   }
 
   @objc static func iconLogoGreenColor() -> UIColor {
-    return UIColor(red: 141.0/255.0, green: 199.0/255.0, blue: 64.0/255.0, alpha: 1.0)
+    UIColor(red: 141.0/255.0, green: 199.0/255.0, blue: 64.0/255.0, alpha: 1.0)
   }
 
   static func cardCreationEnabled() -> Bool {
     return true
   }
+  
+  @objc static func iconColor() -> UIColor {
+    UIApplication.darkModeEnabled ? .white : .black
```

Comment:
I think we should use either system colors for iOS 13+ (e.g., UIColor.label) or create an asset color: right now the app doesn't change button colors when iOS changes dark mode value, rendering buttons invisible:
![Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2021-07-13 at 00 10 15](https://user-images.githubusercontent.com/941531/125343518-97a54d80-e36f-11eb-8ef4-aade4db18d70.png)

Reply:
Good call, updated.

Reply:
Ah I see now, this is going to be a bit tricker since the reader can be in a different mode than the app itself.